sdk/core/openrewrite/rewrite-sample/azure-ai-translation-text-v1/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
```

Comment:
I don't think we should bring in these `azure-ai-translation-text-v1/2` samples into this repo. We should just ensure we have full tests to fully cover all the scenarios in the open rewrite packages.